web/client/plugins/tutorial/preset/dashboard_tutorial.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = [
+    {
+        translation: 'dashboardIntro',
+        selector: '#intro-tutorial'
+    },
+    {
+        translation: 'dashboardNav',
+        selector: '#mapstore-navbar',
+        position: 'bottom',
+        action: {
+            start: {
+                type: 'DASHBOARD:SET_EDITING',
+                editing: false
+            }
+        }
+    },
+    {
+        translation: 'dashboardContainer',
```

Comment:
When you can not edit the tutorial says: 
"You can resize and move dashboards" but you can't. 
We should explain it in a different way.